src/Microsoft.Extensions.RuntimeEnvironment.Sources/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {},
+  "frameworks": {
+    "netstandard1.0": {},
+    "net451": {}
```

Comment:
Yeah, I'm not even sure how it's working now since you use `RuntimeInformation` but don't have it as a dependency. Maybe because we don't ever compile it in this project, but we definitely shouldn't be relying on the consumer having the right dependencies :)